plugins/services/src/js/components/modals/ServiceDestroyModal.js

```diff
@@ -235,6 +236,7 @@ class ServiceDestroyModal extends React.Component {
           onChange={this.handleChangeInputFieldDestroy}
           type="text"
           value={this.state.serviceNameConfirmationValue}
+          autoFocus
```

Comment:
I believe that everything that is part of the browser default behavior should not be tested since browser vendors has put the feats through tests, that being said no need for test on our end but good question though.